akka-docs/src/test/scala/docs/cluster/ClusterDocSpec.scala

```diff
@@ -41,4 +41,14 @@ class ClusterDocSpec extends AkkaSpec(ClusterDocSpec.config) with CompileOnlySpe
     }
   }
 
+  "demonstrate programatic joining to seed nodes" in compileOnlySpec {
+    //#join-seed-nodes
+    import akka.actor.Address
```

Comment:
:+1:, also import `Cluster`?

Reply:
Okay sure, added imports in the new commit https://github.com/akka/akka/pull/24982/commits/90ed27225697416fdc20597f0c96a172eb63ea9d

![2018-04-29_21h01_46](https://user-images.githubusercontent.com/7414320/39406410-4def2ce2-4bf1-11e8-883a-ea7468b501a7.png)
![2018-04-29_21h01_53](https://user-images.githubusercontent.com/7414320/39406412-4f4ca7f4-4bf1-11e8-9d38-ff5b7692d21e.png)